### ISSUE TITLE
ECONFLICT error when multiple sources have different dependencies

### ISSUE THREAD

**Issue by ahmadazimi** [zont/gulp-bower#35]
I found somthing wrong in this library.
When I use `bower update` separately, it ask me what to do when finds a conflict in packages libraries. But in gulp-bower, throw an error and can't handle this exception.

Is this a bug?

```javascript
[08:50:40] bower resolved https://github.com/select2/select2.git#d3c887286d
[08:50:40] 'bower' errored after 1.38 min
[08:50:40] jquery in plugin 'gulp-bower'
Message:
    Unable to find suitable version for jquery
Details:
    code: ECONFLICT
    picks: [object Object],[object Object],[object Object],[object Object],[object Object],[object Object]
[08:50:40] 'default' errored after 1.38 min
[08:50:40] Error: [object Object]
    at formatError (/usr/local/lib/node_modules/gulp/bin/gulp.js:169:10)
    at Gulp.<anonymous> (/usr/local/lib/node_modules/gulp/bin/gulp.js:195:15)
    at Gulp.emit (events.js:117:20)
    at Gulp.Orchestrator._emitTaskDone (/Applications/XAMPP/xamppfiles/htdocs/fwk/node_modules/gulp/node_modules/orchestrator/index.js:264:8)
    at /Applications/XAMPP/xamppfiles/htdocs/fwk/node_modules/gulp/node_modules/orchestrator/index.js:275:23
    at finish (/Applications/XAMPP/xamppfiles/htdocs/fwk/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:21:8)
    at cb (/Applications/XAMPP/xamppfiles/htdocs/fwk/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:29:3)
    at finish (/Applications/XAMPP/xamppfiles/htdocs/fwk/node_modules/run-sequence/index.js:48:5)
    at Gulp.onError (/Applications/XAMPP/xamppfiles/htdocs/fwk/node_modules/run-sequence/index.js:55:4)
    at Gulp.emit (events.js:117:20)
```

**Comment by jfbibeau** [comment]
+1 having this issue here too. Maybe running bower with the --force option through gulp-bower would be a workaround for you?

**Comment by ahmadazimi** [comment]
Running bower with the --force option through gulp-bower fix the problem, But I don't want do that, because I want choose which conflicted item should be select.

**Comment by jfbibeau** [comment]
It would also be nice if the "picks" were displayed in the stack, instead of [object Object], so let the user know what the conflict is...

**Comment by fernandofleury** [comment]
The plugin is missing a `on('promp)` handling. I will work on it right now.

**Comment by jfbibeau** [comment]
That would be awesome :)

**Comment by zont** [comment]
Fixed in #49

**Comment by ahmadazimi** [comment]
:+1:
